### PR TITLE
Integrations to packs - Phase 9 (Dan)

### PR DESCRIPTION
**Pull request by dantavori**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Description
Renamed Devo_v2 pack name to Devo

Moved the following from Legacy to packs:
- integration-Box.yml
- integration-Centreon.yml
- integration-CheckpointFirewall.yml
- integration-cisco-meraki.yml
- integration-CiscoSpark.yml
- integration-CloudLocki.yml
- integration-Cofense-Intelligence.yml
- integration-CrowdStrikeFalconSandbox.yml
- integration-CrowdStrikeHost.yml
- integration-CrowdStrikeIntel.yml
- integration-CuckooSandbox.yml
- integration-cyberark_AIM.yml
- integration-CyberTriage.yml
- integration-Devo.yml
- integration-DigitalShadows.yml
- integration-DNSDB.yml
- integration-DomainTools.yml
- integration-EasyVista.yml
- integration-Endgame.yml
- integration-epo.yml

### CODE REVIEW

File: Packs/Box/pack_metadata.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    "support": "xsoar",
+    "currentVersion": "1.0.0",
+    "author": "Cortex XSOAR",
+    "url": "https://www.paloaltonetworks.com/cortex",
+    "email": "",
+    "categories": [
+        "IT Services"
+    ],
+    "tags": [
+        "Box"
```
Comment (yuvalbenshalom) [inline Packs/Box/pack_metadata.json:13]: remove

File: Packs/Centreon/pack_metadata.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    "support": "xsoar",
+    "currentVersion": "1.0.0",
+    "author": "Cortex XSOAR",
+    "url": "https://www.paloaltonetworks.com/cortex",
+    "email": "",
+    "categories": [
+        "Case Management"
+    ],
+    "tags": [
+        "Centreon"
```
Comment (yuvalbenshalom) [inline Packs/Centreon/pack_metadata.json:13]: remove

File: Packs/CheckpointFirewall/pack_metadata.json [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    "support": "xsoar",
+    "currentVersion": "1.0.0",
+    "author": "Cortex XSOAR",
+    "url": "https://www.paloaltonetworks.com/cortex",
+    "email": "",
+    "categories": [
+        "Network Security"
+    ],
+    "tags": [
+        "Check Point Firewall"
```
Comment (yuvalbenshalom) [inline Packs/CheckpointFirewall/pack_metadata.json:13]: remove

File: Packs/CiscoSpark/pack_metadata.json [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+    "support": "xsoar",
+    "currentVersion": "1.0.0",
+    "author": "Cortex XSOAR",
+    "url": "https://www.paloaltonetworks.com/cortex",
+    "email": "",
+    "categories": [
+        "Messaging"
+    ],
+    "tags": [
+        "Cisco Webex Teams"
```
Comment (yuvalbenshalom) [inline Packs/CiscoSpark/pack_metadata.json:13]: remove

File: Packs/CloudLock/pack_metadata.json [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+    "support": "xsoar",
+    "currentVersion": "1.0.0",
+    "author": "Cortex XSOAR",
+    "url": "https://www.paloaltonetworks.com/cortex",
+    "email": "",
+    "categories": [
+        "Network Security"
+    ],
+    "tags": [
+        "Cisco CloudLock"
```
Comment (yuvalbenshalom) [inline Packs/CloudLock/pack_metadata.json:13]: remove

File: Packs/Cofense-Intelligence/pack_metadata.json [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+    "support": "xsoar",
+    "currentVersion": "1.0.0",
+    "author": "Cortex XSOAR",
+    "url": "https://www.paloaltonetworks.com/cortex",
+    "email": "",
+    "categories": [
+        "Data Enrichment & Threat Intelligence"
+    ],
+    "tags": [
+        "Cofense Intelligence"
```
Comment (yuvalbenshalom) [inline Packs/Cofense-Intelligence/pack_metadata.json:13]: remove

File: Packs/CrowdStrikeFalconSandbox/pack_metadata.json [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+    "support": "xsoar",
+    "currentVersion": "1.0.0",
+    "author": "Cortex XSOAR",
+    "url": "https://www.paloaltonetworks.com/cortex",
+    "email": "",
+    "categories": [
+        "Forensics & Malware Analysis"
+    ],
+    "tags": [
+        "CrowdStrike Falcon Sandbox"
```
Comment (yuvalbenshalom) [inline Packs/CrowdStrikeFalconSandbox/pack_metadata.json:13]: remove

File: Packs/CrowdStrikeHost/pack_metadata.json [hunk 8]
```diff
@@ -0,0 +1,23 @@
...
+    "support": "xsoar",
+    "currentVersion": "1.0.0",
+    "author": "Cortex XSOAR",
+    "url": "https://www.paloaltonetworks.com/cortex",
+    "email": "",
+    "categories": [
+        "Endpoint"
+    ],
+    "tags": [
+        "FalconHost"
```
Comment (yuvalbenshalom) [inline Packs/CrowdStrikeHost/pack_metadata.json:13]: remove

File: Packs/CrowdStrikeIntel/pack_metadata.json [hunk 9]
```diff
@@ -0,0 +1,23 @@
...
+    "support": "xsoar",
+    "currentVersion": "1.0.0",
+    "author": "Cortex XSOAR",
+    "url": "https://www.paloaltonetworks.com/cortex",
+    "email": "",
+    "categories": [
+        "Data Enrichment & Threat Intelligence"
+    ],
+    "tags": [
+        "CrowdStrike Falcon Intel"
```
Comment (yuvalbenshalom) [inline Packs/CrowdStrikeIntel/pack_metadata.json:13]: remove

File: Packs/CuckooSandbox/pack_metadata.json [hunk 10]
```diff
@@ -0,0 +1,23 @@
...
+    "support": "xsoar",
+    "currentVersion": "1.0.0",
+    "author": "Cortex XSOAR",
+    "url": "https://www.paloaltonetworks.com/cortex",
+    "email": "",
+    "categories": [
+        "Forensics & Malware Analysis"
+    ],
+    "tags": [
+        "Cuckoo Sandbox"
```
Comment (yuvalbenshalom) [inline Packs/CuckooSandbox/pack_metadata.json:13]: remove

File: Packs/CyberTriage/pack_metadata.json [hunk 11]
```diff
@@ -0,0 +1,23 @@
...
+    "support": "xsoar",
+    "currentVersion": "1.0.0",
+    "author": "Cortex XSOAR",
+    "url": "https://www.paloaltonetworks.com/cortex",
+    "email": "",
+    "categories": [
+        "Endpoint"
+    ],
+    "tags": [
+        "Cyber Triage"
```
Comment (yuvalbenshalom) [inline Packs/CyberTriage/pack_metadata.json:13]: remove

File: Packs/DNSDB/pack_metadata.json [hunk 12]
```diff
@@ -0,0 +1,23 @@
...
+    "support": "xsoar",
+    "currentVersion": "1.0.0",
+    "author": "Cortex XSOAR",
+    "url": "https://www.paloaltonetworks.com/cortex",
+    "email": "",
+    "categories": [
+        "Data Enrichment & Threat Intelligence"
+    ],
+    "tags": [
+        "Farsight DNSDB"
```
Comment (yuvalbenshalom) [inline Packs/DNSDB/pack_metadata.json:13]: remove

File: Packs/DigitalShadows/pack_metadata.json [hunk 13]
```diff
@@ -0,0 +1,23 @@
...
+    "support": "xsoar",
+    "currentVersion": "1.0.0",
+    "author": "Cortex XSOAR",
+    "url": "https://www.paloaltonetworks.com/cortex",
+    "email": "",
+    "categories": [
+        "Data Enrichment & Threat Intelligence"
+    ],
+    "tags": [
+        "Digital Shadows"
```
Comment (yuvalbenshalom) [inline Packs/DigitalShadows/pack_metadata.json:13]: remove

File: Packs/DomainTools/pack_metadata.json [hunk 14]
```diff
@@ -0,0 +1,23 @@
...
+    "support": "xsoar",
+    "currentVersion": "1.0.0",
+    "author": "Cortex XSOAR",
+    "url": "https://www.paloaltonetworks.com/cortex",
+    "email": "",
+    "categories": [
+        "Data Enrichment & Threat Intelligence"
+    ],
+    "tags": [
+        "DomainTools"
```
Comment (yuvalbenshalom) [inline Packs/DomainTools/pack_metadata.json:13]: remove

File: Packs/EasyVista/pack_metadata.json [hunk 15]
```diff
@@ -0,0 +1,23 @@
...
+    "support": "xsoar",
+    "currentVersion": "1.0.0",
+    "author": "Cortex XSOAR",
+    "url": "https://www.paloaltonetworks.com/cortex",
+    "email": "",
+    "categories": [
+        "IT Services"
+    ],
+    "tags": [
+        "EasyVista"
```
Comment (yuvalbenshalom) [inline Packs/EasyVista/pack_metadata.json:13]: remove

File: Packs/Endgame/pack_metadata.json [hunk 16]
```diff
@@ -0,0 +1,23 @@
...
+    "support": "xsoar",
+    "currentVersion": "1.0.0",
+    "author": "Cortex XSOAR",
+    "url": "https://www.paloaltonetworks.com/cortex",
+    "email": "",
+    "categories": [
+        "Endpoint"
+    ],
+    "tags": [
+        "Endgame"
```
Comment (yuvalbenshalom) [inline Packs/Endgame/pack_metadata.json:13]: remove

File: Packs/cisco-meraki/pack_metadata.json [hunk 17]
```diff
@@ -0,0 +1,23 @@
...
+    "support": "xsoar",
+    "currentVersion": "1.0.0",
+    "author": "Cortex XSOAR",
+    "url": "https://www.paloaltonetworks.com/cortex",
+    "email": "",
+    "categories": [
+        "IT Services"
+    ],
+    "tags": [
+        "Cisco Meraki"
```
Comment (yuvalbenshalom) [inline Packs/cisco-meraki/pack_metadata.json:13]: remove

File: Packs/cyberark_AIM/pack_metadata.json [hunk 18]
```diff
@@ -0,0 +1,23 @@
...
+    "support": "xsoar",
+    "currentVersion": "1.0.0",
+    "author": "Cortex XSOAR",
+    "url": "https://www.paloaltonetworks.com/cortex",
+    "email": "",
+    "categories": [
+        "Authentication"
+    ],
+    "tags": [
+        "CyberArk AIM"
```
Comment (yuvalbenshalom) [inline Packs/cyberark_AIM/pack_metadata.json:13]: remove

File: Packs/epo/pack_metadata.json [hunk 19]
```diff
@@ -0,0 +1,23 @@
...
+    "support": "xsoar",
+    "currentVersion": "1.0.0",
+    "author": "Cortex XSOAR",
+    "url": "https://www.paloaltonetworks.com/cortex",
+    "email": "",
+    "categories": [
+        "Endpoint"
+    ],
+    "tags": [
+        "McAfee ePO"
```
Comment (yuvalbenshalom) [inline Packs/epo/pack_metadata.json:13]: remove

**Comment by yuvalbenshalom** [comment]
test filter still failing.
check conflicts.

**Comment by dantavori** [comment]
@yuvalbenshalom I used demisto-sdk's format for a lot of yml files and it caused schema issues in all of them. I saw @liorblob opened an issue about it.

**Pull request merged**